Resources/Prototypes/Entities/Objects/Tools/jetpacks.yml

```diff
@@ -0,0 +1,293 @@
+      moles:
+        - 22.6293856
+
+#Empty black
+- type: entity
+  id: JetpackBlack
+  parent: BaseJetpack
+  name: jetpack
+  suffix: Empty
+  components:
+  - type: Sprite
+    sprite: Objects/Tanks/Jetpacks/black.rsi
+  - type: Clothing
+    sprite: Objects/Tanks/Jetpacks/black.rsi
+    Slots:
+      - Back
+
+# Filled black
+- type: entity
+  id: JetpackBlackFilled
+  parent: JetpackBlack
+  name: jetpack
+  suffix: Filled
+  components:
+  - type: GasTank
+    outputPressure: 21.27825
+    air:
+      volume: 70
+      temperature: 293.15
+      moles:
+        - 22.6293856
+
+#Empty captain
+- type: entity
+  id: JetpackCaptain
+  parent: BaseJetpack
+  name: captain's jetpack
+  suffix: Empty
+  components:
+  - type: Sprite
+    sprite: Objects/Tanks/Jetpacks/captain.rsi
+  - type: Clothing
+    size: 30
+    sprite: Objects/Tanks/Jetpacks/captain.rsi
+    Slots:
+      - Back
+
+# Filled captain
+- type: entity
+  id: JetpackCaptainFilled
+  parent: JetpackCaptain
+  name: captain's jetpack
+  suffix: Filled
+  components:
+  - type: GasTank
+    outputPressure: 21.27825
+    air:
+      volume: 70
+      temperature: 293.15
+      moles:
+        - 22.6293856
+
+#Empty improvised
+- type: entity
+  id: JetpackImprovised
+  parent: BaseJetpack
+  name: improvised jetpack
+  suffix: Empty
+  components:
+  - type: Sprite
+    sprite: Objects/Tanks/Jetpacks/improvised.rsi
+  - type: Clothing
+    sprite: Objects/Tanks/Jetpacks/improvised.rsi
+    Slots:
+      - Back
+
+# Filled improvised
+- type: entity
+  id: JetpackImprovisedFilled
+  parent: JetpackImprovised
+  name: improvised jetpack
+  suffix: Filled
+  components:
+    - type: GasTank
+      outputPressure: 21.27825
+      air:
+        volume: 70
+        temperature: 293.15
+        moles:
+          - 22.6293856
+
+#Empty mini
+- type: entity
+  id: JetpackMini
+  parent: BaseJetpack
+  name: mini jetpack
+  suffix: Empty
+  components:
+    - type: Sprite
+      sprite: Objects/Tanks/Jetpacks/mini.rsi
+    - type: Clothing
+      size: 50
+      sprite: Objects/Tanks/Jetpacks/mini.rsi
+      Slots:
+        - Back
+
+# Filled mini
+- type: entity
+  id: JetpackMiniFilled
+  parent: JetpackMini
+  name: mini jetpack
+  suffix: Filled
+  components:
+  - type: GasTank
+    outputPressure: 21.27825
+    air:
+      temperature: 293.15
+      volume: 2
+      moles:
+      - 0.323460326
+
+#Empty mining
+- type: entity
+  id: JetpackMining
+  parent: BaseJetpack
+  name: mining jetpack
+  suffix: Empty
+  components:
+    - type: Sprite
+      sprite: Objects/Tanks/Jetpacks/mining.rsi
+    - type: Clothing
+      sprite: Objects/Tanks/Jetpacks/mining.rsi
+      Slots:
+        - Back
+
+# Filled mining
+- type: entity
+  id: JetpackMiningFilled
+  parent: JetpackMining
+  name: mining jetpack
+  suffix: Filled
+  components:
+  - type: GasTank
+    outputPressure: 21.27825
+    air:
+      volume: 70
+      temperature: 293.15
+      moles:
+        - 22.6293856
+
```

Comment:
remove. improvised jetpacks are unneeded bc of throwing+extinguishers, mining shouldn't get jetpacks